test/org/sagebionetworks/bridge/play/controllers/ConsentControllerMockedTest.java

```diff
@@ -634,52 +376,35 @@ public void dataSharingSuspendedUpdatesSession() throws Exception {
         assertFalse(node.get("dataSharing").asBoolean());
 
         verify(account).setSharingScope(SharingScope.NO_SHARING);
+        
+        verify(accountDao).editAccount(eq(TestConstants.TEST_STUDY), eq(HEALTH_CODE),
+                accountConsumerCaptor.capture());
+        
+        Consumer<Account> accountConsumer = accountConsumerCaptor.getValue();
+        Account updatedAccount = Account.create();
+        accountConsumer.accept(updatedAccount);
+        assertEquals(SharingScope.NO_SHARING, updatedAccount.getSharingScope());
```

Comment:
This block doesn't actually do anything. The default sharing scope is no_sharing. This is also redundant with the block above, which already verifies that account.setSharingScope() was called.

Reply:
Sorry you asked about testing the lambda... I actually am not sure how to verify the lambda functionality beyond what the test originally did... verified that the setter on the account was called with the correct sharing status. Removed.